### PR TITLE
Move amazon tab to configuration tab of channel #7837

### PR DESCRIPTION
**Pull request by vishu1993**

### CODE REVIEW

File: view/sale_channel_form.xml [hunk 1]
```diff
@@ -12,19 +11,25 @@
             <label name="amazon_secret_key"/>
             <field name="amazon_secret_key" widget="password"/>
             <newline/>
-            <separator id="import_export_time" string="Import / Export Time" colspan="2"/>
+        </group>
+    </xpath>
+    <xpath expr="/form/notebook/page[@id='configuration']/notebook/page[@id='taxes']" position="after">
+        <page id="last_import_export_time" string="Import / Export Time" states="{'invisible': Not(Eval('source') == 'amazon_mws')}" >
+            <separator id="last_import_export" string="Import / Export Time" colspan="2"/>
```
Comment (priyankarani) [inline view/sale_channel_form.xml]: separator is not needed if its a new tab

**Comment by vishu1993** [comment]
amazon2

File: product.py [hunk 2]
```diff
@@ -123,8 +123,6 @@ def extract_product_values_from_amazon_data(cls, product_attributes):
             'default_uom': amazon_channel.default_uom.id,
             'salable': True,
             'sale_uom': amazon_channel.default_uom.id,
-            'account_expense': amazon_channel.default_account_expense.id,
-            'account_revenue': amazon_channel.default_account_revenue.id,
         }
```
Comment (priyankarani) [inline product.py]: no need. just rebase you code

File: tests/test_base.py [hunk 3]
```diff
@@ -261,7 +258,7 @@ def setup_defaults(self):
         # TODO: This should work without creating new properties
         self.Property.create([{
             'value': 'account.account' + ',' + str(
-                self.sale_channel.default_account_revenue.id
+                self.get_account_by_kind('revenue')
```
Comment (priyankarani) [inline tests/test_base.py]: remove this change and rebase

**Pull request merged**